libs/sdk-ui-filters/src/AttributeFilterLoader/internal/store/state.ts

```diff
@@ -15,21 +14,30 @@ import {
  * @internal
  */
 export interface AttributeFilterState {
-    // Attribute filter component state
+    // Effective attribute filter -> move this to selector?
```

Comment:
Yes, I would make this a selector, otherwise there will be duplicities in the information about the selection etc.